src/types/role.ts

```diff
@@ -15,4 +15,12 @@ export interface RoleData {
   managed: boolean;
   /** whether this role is mentionable */
   mentionable: boolean;
+  /** Certain roles may have tags that allow you to determine if this role is related to a bot, an integration, or the booster role. */
+  tags: RoleTags | null;
+}
+
+export interface RoleTags {
+  bot_id?: string;
+  premium_subscriber?: null;
```

Comment:
Is this meant to be ?: boolean | null;

Reply:
@Skillz4Killz no, like I mentioned earlier, it may exist (sent by the API) with the value `null` or may not exist (not sent by API) at all.